src/csharp/PesterTests/InvocationResultTests.cs

```diff
@@ -0,0 +1,42 @@
+namespace PesterTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Management.Automation;
+    using Pester;
+    using Xunit;
+
+    /// <summary>
+    /// Invocation Result Tests
+    /// </summary>
+    public class InvocationResultTests
+    {
+        public readonly InvocationResult invocationResult;
+
+        public InvocationResultTests()
+        {
+            var success = true;
+            var errorRecords = new List<ErrorRecord>();
+            var standardOutput = new Object();
+
+            InvocationResult testInvocationResult =
+                new InvocationResult(success, errorRecords, standardOutput);
+            Assert.NotNull (testInvocationResult);
```

Comment:
Please no spaces after method name.

Reply:
Will see where the autoformatter is falling down here.

Reply:
Had prettier enabled from another PR.